emission/analysis/userinput/matcher.py

```diff
@@ -233,6 +236,7 @@ def create_confirmed_entry(ts, tce, confirmed_key, input_key_list):
             tce["data"]["cleaned_trip"])
         confirmed_object_data['inferred_section_summary'] = get_section_summary(ts, cleaned_trip, "analysis/inferred_section")
         confirmed_object_data['cleaned_section_summary'] = get_section_summary(ts, cleaned_trip, "analysis/cleaned_section")
+        confirmed_object_data['ble_sensed_summary'] = get_section_summary(ts, cleaned_trip, "analysis/inferred_section", mode_prop="ble_sensed_mode")
```

Comment:
I would assume that you would want to use `analysis/cleaned_section` here since the inferred section is not based on raw/smoothed data from sensors, but by running an ML pipeline.

Reply:
Sensed data goes with sensor data, makes sense
TODO